vim/autoload/vlime/overlay/slimv.vim

```diff
@@ -492,3 +492,10 @@ function! s:DialogUntraceAllComplete(conn, result)
         endfor
     endif
 endfunction
+
+
+function! SlimvEval(data)
```

Comment:
I'm not sure about the function name. Maybe it should be put into Vlime's namespace, and called `vlime#overlay#slimv#SlimvEval` instead?

Reply:
Sorry about the delay... didn't notice that any sooner.

Yeah, want to provide a PR?